### PR TITLE
event.which differences...

### PR DESCRIPTION
**Pull request by CrandellWS**
This better shows a difference that is not well documented...It is believed that by showing the difference in the sample people will more quickly identify the situation...

### CODE REVIEW

**Comment by arthurvr** [comment]
ping @CrandellWS about the above.

**Comment by arthurvr** [comment]
Looks like now the CLA is signed now and the code styling is addressed, thanks @CrandellWS!

File: entries/event.which.xml [hunk 1]
```diff
@@ -13,7 +13,10 @@
     <desc>Log which key was depressed.</desc>
     <code><![CDATA[
 $( "#whichkey" ).on( "keydown", function( event ) {
-  $( "#log" ).html( event.type + ": " +  event.which );
+    $( "#log" ).append( event.type + ": " + event.which + "<br>" );
+});
+$( "#whichkey" ).on( "keypress", function( event ) {
+    $( "#log" ).append( event.type + ": " + event.which + "<br>" );
```
Comment (arthurvr) [inline entries/event.which.xml]: We use 2 spaces indentation for examples.

Comment (scottgonzalez) [inline entries/event.which.xml]: Actually, indentation should always use tabs. The build process converts to spaces.

Comment (arthurvr) [inline entries/event.which.xml]: @scottgonzalez Nearly no example does that, so I'd rather vote for consistency.

Comment (arthurvr) [inline entries/event.which.xml]: ping @CrandellWS

Comment (arthurvr) [inline entries/event.which.xml]: @CrandellWS still interested in taking this up ^

Comment (arthurvr) [inline entries/event.which.xml]: ping @CrandellWS

Comment (CrandellWS) [inline entries/event.which.xml]: @arthurvr ??

Comment (arthurvr) [inline entries/event.which.xml]: @CrandellWS There are updates that need to happen to this PR. Can you make them?

Comment (CrandellWS) [inline entries/event.which.xml]: I am confused as to what you are asking for...which why I gave you write access to the repository...

**Pull request closed without merging**